packages/xo-server-audit/src/index.js

```diff
@@ -114,6 +115,27 @@ class Db extends Storage {
   getLastId() {
     return this.get(LAST_ID)
   }
+
+  async clean() {
+    const db = this._db
+
+    // delete first so that a new chain can be constructed even if anything else fails
+    await db.del(LAST_ID)
```

Comment:
What's the behavior if it does not exist?

Reply:
Why should we clean the DB if there is no records?

Reply:
What if it does not exist but there are still some entries left?

Anyway, I just tried and it does not throw in case of missing entry.